index.js

```diff
@@ -60,12 +56,49 @@ const gitTasks = opts => {
 	return new Listr(tasks);
 };
 
+const prerequisiteTasks = newVersion => {
+	return new Listr([
+		{
+			title: 'Check Node.js and npm version',
+			skip: () => semver.lt(process.version, '6.0.0'),
+			task: () => execa.stdout('npm', ['version', '--json']).then(json => {
+				const versions = JSON.parse(json);
+				if (!semver.satisfies(versions.npm, '>=2.15.8 <3.0.0 || >=3.10.1')) {
+					return Promise.reject(new Error(`npm@${versions.npm} has known issues publishing when running Node.js 6. Please upgrade npm or downgrade Node and publish again. https://github.com/npm/npm/issues/5082`));
+				}
+			})
+		},
+		{
+			title: 'Fetch remote changes',
+			task: () => execa('git', ['fetch'])
+		},
+		{
+			title: 'Check git tag existance',
+			task: () => execa.stdout('git', ['rev-parse', '--quiet', '--verify', `refs/tags/v${newVersion}`])
+				.then(
+					output => {
+						if (output) {
+							throw new Error(`Git tag \`v${newVersion}\` already exists.`);
+						}
+					},
+					err => {
+						if (err.stdout !== '' || err.stderr !== '') {
+							throw err;
+						}
```

Comment:
That's what I thought as well. It doesn't output errors, but it still exists with status code 1. If you run that command in the terminal, it looks like this.

> $ git rev-parse -q --verify refs/tags/v0.4.0
> 8cda9fb6af972a74641bd27059de0a5c250f4d0b
> 
> $ git rev-parse -q --verify refs/tags/v1.0.0
> <br>

But when running it with `execa`, this is what the error object looks like

```json
{
    "killed": false,
    "code": 1,
    "signal": null,
    "cmd": "git rev-parse --quiet --verify refs/tags/v1.0.0",
    "stdout": "",
    "stderr": ""
}
```

And now without `-q --verify`

```json
{
    "killed": false,
    "code": 128,
    "signal": null,
    "cmd": "git rev-parse refs/tags/v1.0.0",
    "stdout": "refs/tags/v1.0.0\n",
    "stderr": "fatal: ambiguous argument 'refs/tags/v1.0.0': unknown revision or path not in the working tree.\nUse '--' to separate paths from revisions, like this:\n'git <command> [<revision>...] -- [<file>...]'\n"
}
```

The reason they exit with `1` I guess is to be able to write commands that looks like this

> $ git rev-parse -q --verify refs/tags/v1.0.0 && echo "exists"

Not sure though, just guessing here.